src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="index.html">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li>
+            <a class="nav-link is-active" href="#Apple">Apple</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#Samsung">Samsung</a>
+          </li>
```

Comment:
```suggestion
    <header class="header">
        <a href="index.html" class="logo">
          <img src="images/logo.png" alt="logo">
        </a>
        
      <nav class="nav">
        <ul class="nav-list">
          <li>
            <a class="nav-link is-active" href="#Apple">
              Apple
            </a>
          </li>
          
          <li>
            <a class="nav-link" href="#Samsung">
              Samsung
            </a>
          </li>
```
- you dont need extra div
- fix code style